docs/changelog.rst

```diff
@@ -1,6 +1,16 @@
 Changelog
 ~~~~~~~~~
 
+0.4.0 (October 2023)
+--------------------
+
+* API Update. Configuration of ``MPS`` simulation parameters is now done via ``ConfigMPS``.
+* Added a ``value_of_zero`` parameter to ``ConfigMPS`` for the user to indicate the threshold below which numbers are so small that can be interpreted as zero.
+* Added a logger to MPS methods. Use it by setting ``loglevel`` in ``ConfigMPS``.
+* Improved performance of contraction across ``MPS`` methods by hardcoding the contraction paths.
+* Fixed a bug that caused more MPS canonicalisation than strictly required.
+* Fixed a bug where simulate would not apply the last batch of gates when using ``MPSxMPO``.
```

Comment:
Should 'simulate' be quoted (`simulate`) in the last line?

Reply:
Yup!